client/src/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+var Test = () => (
+  <div>super cool test</div>
+)
```

Comment:
You should configure your linter immediately so your reviewers don't have to spend time looking over missing `;`s.